src/components/Bar/examples/custom-colors.jsx

```diff
@@ -18,11 +18,11 @@ export default React.createClass({
 		return (
 			<div>
 				<svg {...svgProps}>
-					<Bar {...pointProps} color='#f80' />
+					<Bar {...pointProps} color="#f80" />
```

Comment:
This is what we would do way back when we writing raw html inside of JS in HBUI, so it kind of makes sense I guess.